tailwind.config.js

```diff
@@ -91,7 +97,10 @@ module.exports = {
       }
     },
   },
-  variants: {},
+  variants: {
+    backgroundColor: ['responsive', 'hover', 'focus', 'active'],
+    boxShadow: ['responsive', 'hover', 'focus', 'active']
+  },
```

Comment:
Needed to add these in to allow `hover`, `focus`, `active` states (they are turned off by default)